bindingcollectionadapter/src/main/java/me/tatarka/bindingcollectionadapter2/itembindings/OnItemBindClass.java

```diff
@@ -44,6 +74,26 @@ public int itemTypeCount() {
 
     @Override
     public void onItemBind(ItemBinding itemBinding, int position, T item) {
+        itemBinding.clearExtra();
+        if (itemExtraBindingMap != null && !itemExtraBindingMap.isEmpty()) {
+            for (int i = 0; i < itemExtraBindingMap.size(); i++) {
+                Class<? extends T> key = itemExtraBindingMap.keyAt(i);
+                if (key.isInstance(item)) {
```

Comment:
>>What's the use case of this? If you want different extra's per item, why not include that stuff in the item itself?

Example:

I have merged list:

    public final MergeObservableList<Object> members = new MergeObservableList<>()
            .insertItem(new AddButton())
            .insertList(speakers)
            .insertList(listeners);

I want to highlight last speaker item:

    public final OnItemBind<Object> binding = new OnItemBindClassMap<>()
            .map(Card.class, BR.item, R.layout.card_list_item)
            .mapExtra(Card.class, BR.highlight, o -> Utils.isLast(speakers, o))
            .map(AddButton.class, BR.item, R.layout.add_button);